examples/guess-the-sketch/src/app.py

```diff
@@ -178,37 +187,43 @@ def handle_message(data):
     round = int(data['round'])
 
     logger.debug('Player %s: Received guess %s for opponent %s: %s', player_id, str(round), oppontent_id, message)
+    encoded_image = "placeholder"
+    if LIMITED_PROMPTS == "true":
+        logger.info("I'm here instead")
```

Comment:
please update with a more informative debugging message.

Reply:
oops, now you know my secret debug code 😄